docs/source/quickstart.mdx

```diff
@@ -19,17 +19,18 @@ The following code illustrates the steps above.
 # 0. imports
 import torch
 from transformers import GPT2Tokenizer
-from trl.model import AutoModelWithValueModel, respond_to_batch
```

Comment:
then we should also use `ppo_trainer.generate` instead of `respond_to_batch` :)

Reply:
`respond_to_batch` has been imported below, I think we can let the script as it is! :D